src/server/db/models/github-user.js

```diff
@@ -9,7 +9,7 @@ import db from '../index';
  *   updated_at: update date
  *   last_login_at: last login date
  */
-export const GitHubUser = db.Model.extend({
+db.model('GitHubUser', {
```

Comment:
I was thinking about a pattern that would solve the issue of circular dependency between models and db.

What I think would help would be to wrap model declarations in function that will get db in param (rather then importing it directly).

```js
export default function registerGitHubUser(db) {
  db.model('GitHubUser', { ... });
};
```

It's not perfect, but it would allow models to be properly imported in `db`:

```js
import * as models from `./models`;

// db module code

models.registerGitHubUser(db);

export default db;
```

Still `registerGitHubUser` has side-effects (changed db that is passed to it), but at least circular dependency is removed and relation between model and db is a bit more explicit.

But I leave it to you to decide what pattern you prefer.